src/App/App.js

```diff
@@ -30,45 +33,46 @@ function App() {
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
     .then((res) => {
       if (!res.ok) {
-        console.log("error");
+        displayError(400);
       } else {
         return res.json();
       } 
     })
     .then((data) => setSelectedMovie(data.movie))
+    .catch(err => displayError(500))
   }
 
-
   function showDesc(evt) {
     getSingleMovie(evt.target.id);
     setIsMovieSelected(true);
   }
+
+  function displayHomePage() {
+    setIsMovieSelected(false)
+    setSelectedMovie({})
+  }
+
+  function displayError(responseLevel) {
+    setIsError(true)
```

Comment:
Do we need `isError` to exist? Can we just use `responseLevel` to know if there is an error? i.e. if `responseLevel` is `null`, we know there is no error.